samples/SkiaAwtSample/build.gradle.kts

```
@@ -39,6 +40,7 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-swing:1.5.0")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.5.0")
     implementation("org.jetbrains.skiko:skiko-awt-runtime-$target:$version")
+    implementation("com.jetbrains:jbr-api:1.4.0")
```

```suggestion
    compileOnly("com.jetbrains:jbr-api:1.4.0")
```

We should make it optional (so, it won't be added transitively), and check in runtime if it is included

---

This purpose of this library to check: 
1. Is JBR available.
2. Is certain functionality available if JBR?
3. Provide a wrapper class over this functionality if avaliable?

---

Okay to have it uincoditionally if we publish [to Maven Central](https://github.com/JetBrains/skiko/pull/1031#discussion_r1978955685)